interface/patient_file/front_payment.php

```diff
@@ -35,1302 +29,1190 @@
-                }
-            }
-            if (flgempty) {
-                alert("<?php echo xls('A Payment is Required!. Please input a payment line item entry.') ?>");
-                return false;
-            }
-            // continue validation.
-            if (((document.getElementById('form_method').options[document.getElementById('form_method').selectedIndex].value == 'check_payment' ||
-                    document.getElementById('form_method').options[document.getElementById('form_method').selectedIndex].value == 'bank_draft') &&
-                    document.getElementById('check_number').value == '')) {
-                alert("<?php echo addslashes(xl('Please Fill the Check/Ref Number')) ?>");
-                document.getElementById('check_number').focus();
-                return false;
-            }
+<link rel="stylesheet" type="text/css" href="../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
+<script type="text/javascript" src="../../library/textformat.js"></script>
 
-            if (document.getElementById('radio_type_of_payment_self1').checked == false &&
-                document.getElementById('radio_type_of_payment1').checked == false &&
-                document.getElementById('radio_type_of_payment2').checked == false &&
-                document.getElementById('radio_type_of_payment4').checked == false) {
-                alert("<?php echo addslashes(xl('Please Select Type Of Payment.')) ?>");
-                return false;
-            }
+<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-3-2/index.js"></script>
 
-            if (document.getElementById('radio_type_of_payment_self1').checked == true ||
-                document.getElementById('radio_type_of_payment1').checked == true) {
-                for (var i = 0; i < f.elements.length; ++i) {
-                    var elem = f.elements[i];
-                    var ename = elem.name;
-                    if (ename.indexOf('form_upay[0') == 0) //Today is this text box.
-                    {
-                        if (elem.value * 1 > 0) {//A warning message, if the amount is posted with out encounter.
-                            if (confirm("<?php echo addslashes(xlt('If patient has appointment click OK to create encounter otherwise, cancel this and then create an encounter for today visit.')) ?>")) {
-                                ok = 1;
-                            }
-                            else {
-                                elem.focus();
-                                return false;
-                            }
-                        }
-                        break;
-                    }
-                }
-            }
+<script type="text/javascript" src="../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
+<script type="text/javascript" src="../../library/js/jquery.easydrag.handler.beta2.js"></script>
```

Comment:
Pretty sure above block of code is old. Again concerning that this commit is using old files and likely removing recent code changes.